src/collections/_documentation/error-reporting/configuration/drain-example/native.md

```diff
@@ -6,3 +6,6 @@ sentry_shutdown();
 
 Calling `sentry_shutdown()` before exiting the application is critical to avoid
 data loss.
+
+After shutdown the client cannot be used any more so make sure to only do that
+right before you shut down the application.
```

Comment:
This is moved in here from the global context with no changes.

As for the general contents of this file, @jan-auer @Swatinem is there anything else you'd like to add?
Is this still accurate and reflecting the best practice?